apps/web/src/components/Home/FlashSale/index.tsx

```diff
@@ -0,0 +1,273 @@
+                    <div className='flex mb-3 md:mb-3.5 pb-0'>
+                        <a className="group relative flex h-50 overflow-hidden">
+                            <Image className="bg-gray-300 object-cover rounded-s-md rounded-md transition duration-150 ease-linear transform group-hover:scale-105" src={flash1} />
+                        </a>
+                    </div>
+
+                    <div className='w-full overflow-hidden p-2 ps-0'>
+                        <h2 className='truncate mb-1 font-semibold md:mb-1.5 text-sm sm:text-base md:text-sm lg:text-base xl:text-lg text-heading'>
+                            Adidas Shoes Black
+                        </h2>
+                        <p className='text-body text-xs lg:text-sm leading-normal xl:leading-relaxed max-w-[250px] truncate'>
+                            Men Black top sleeveless gown
+                        </p>
+                        <div className='font-semibold text-sm sm:text-base mt-1.5 space-s-2 sm:text-xl md:text-base lg:text-xl md:mt-2.5 2xl:mt-3 text-heading'>
+                            <span className='inline-block false'>$45.00</span>
+                            <del className='sm:text-base font-normal text-gray-800 pl-3'>$99.99</del>
+                        </div>
+                    </div>
+                </div>
+
+                <div className="col-start-2 row-start-1">
+                    <div className="group box-border overflow-hidden flex rounded-md cursor-pointer bg-white pe-0 md:pb-1 flex-col items-start">
+                        <div className='flex mb-3 md:mb-3.5 pb-0'>
+                            <a className="group relative flex h-50 overflow-hidden">
+                                <Image className="bg-gray-300 object-cover rounded-s-md rounded-md transition duration-150 ease-linear transform group-hover:scale-105" src={flash2} />
+                            </a>
+                        </div>
+
+                        <div className='w-full overflow-hidden p-2 ps-0'>
+                            <h2 className='truncate mb-1 font-semibold md:mb-1.5 text-sm sm:text-base md:text-sm lg:text-base xl:text-lg text-heading'>
+                                Armani Wide-Leg Trousers
+                            </h2>
+                            <p className='text-body text-xs lg:text-sm leading-normal xl:leading-relaxed max-w-[250px] truncate'>
+                                Monochrome elegance.
+                            </p>
+                            <div className='font-semibold text-sm sm:text-base mt-1.5 space-s-2 sm:text-xl md:text-base lg:text-xl md:mt-2.5 2xl:mt-3 text-heading'>
+                                <span className='inline-block false'>$12.00</span>
+                                <del className='sm:text-base font-normal text-gray-800 pl-3'>$16.00</del>
+                            </div>
+                        </div>
+                    </div>
+                </div>
```

Comment:
I think we can wrap this into reusable component, we could just pass it props for the `name, image, price, description, etc` what do you think?

Reply:
yeah, i will see i can do